strapi/strapi
# [feat]: feature flags

### What does it do?

Introduces a new configuration file: features.(j|t)s. By integrating this configuration into our strapi.config, we gain the ability to determine whether to include the content-releases plugin in our application based on an environment variable. This could be used by any new feature we develop. It's also passed to the admin configuration so future flags can be checked via `window.strapi.future.isEnabled('contentReleases')`.

### Why is it needed?

This allows us to introduce new unstable features into our main branch without impacting user experience. It also facilitates early feedback from users interested in exploring these unstable features that are still in development.

### How to test it?

1. On the getstarted app, make sure to add `STRAPI_FEATURES_FUTURE_CONTENTRELEASES=true` to the .env file.
2. Run your application, you should have the unstable Content-Releases feature enabled
